.gitignore

```diff
@@ -61,7 +61,7 @@ instance/
 
 # Sphinx documentation
 docs/_build/
-docs/_static/
+docs/source/_static/
```

Comment:
```suggestion
docs/_static/
```

Reply:
this is  **docs/source/_static/** the path that I get when I build the project.

Reply:
The docs are not located in `docs/source` anymore, are you sure you ran the docs build from the latest version?

Reply:
yes,well the docs are not located in  `docs/source` but the build is 
![Screenshot from 2024-08-27 16-21-53](https://github.com/user-attachments/assets/eb538db5-bca2-4ff3-8d94-4ad183c2f536)

Reply:
Ah I see what's wrong. We should fix it here: https://github.com/QuantStack/ipyopenlayers/blob/main/webpack.config.js#L78

Reply:
okay thanks, I'll get it done.